content/en/service-mesh.md

```diff
@@ -26,3 +26,16 @@ Service meshes add reliability, observability, and security features
 uniformly across all services across a cluster without requiring code changes. 
 Before service meshes, that functionality had to be encoded into every single service, 
 becoming a potential source of bugs and technical debt.
+
+In the [Sidecar Container] (/sidecar-container/) model, each microservice pod is paired with its own proxy.
+This per-service proxy intercepts L3-L7 traffic, enforcing load-balancing, mutual TLS, 
+tracing and metrics on a fine-grained basis.
+While this approach offers maximal policy flexibility and service-specific routing,
+it also incurs additional CPU and memory overhead for every proxy and increases operational complexity as the mesh grows.
+
+A **Sidecarless** design, on the other hand,
+runs data-plane logic directly in the Linux kernel and combines mesh functionality into a small group of host-level agents that make use of kernel features like [eBPF] (/ebpf/).
+By doing away with per-pod proxies, this method drastically reduces resource usage and removes unnecessary network hops,
+which lowers latency and boosts performance.
```

Comment:
The line break here is strange.

Reply:
I agree. Semantic breaks wise, I think it could go a couple ways, above is one, alternatively:

```
By doing away with per-pod proxies,
this method drastically reduces resource usage and removes unnecessary network hops,
which lowers latency and boosts performance.
```

or, maybe just put it all on one line:

```
By doing away with per-pod proxies, this method drastically reduces resource usage and removes unnecessary network hops, which lowers latency and boosts performance.

Reply:
one line would be most common one based on the semantic, but based on #3492, first is fit, i'll make changes and fix the file once and update the PR by eod.